<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataImport</class>
 <widget class="QMainWindow" name="dataImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qrc.qrc">
    <normaloff>:/src/exp.jpg</normaloff>:/src/exp.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(76,117,136);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_refer">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit
{
border:2px solid white;
background-color: rgb(18, 204, 209);
border-radius:5px;
font:75 11pt &quot;仿宋&quot;;
color:rgb(52,101,64);
}
QLineEdit:hover
{
background-color: rgb(238, 238, 236);
}
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refer2Btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 255, 127);</string>
            </property>
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView_Tables"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="submitBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 255, 127);</string>
            </property>
            <property name="text">
             <string>加载</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="commitBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 255, 127);</string>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="text">
               <string>导入文件：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_imp">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit
{
border:2px solid white;
background-color: rgb(18, 204, 209);
border-radius:5px;
font:75 11pt &quot;仿宋&quot;;
color:rgb(52,101,64);
}
QLineEdit:hover
{
background-color: rgb(238, 238, 236);
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="minimumSize">
               <size>
                <width>55</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>31</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>格式</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>*.sql</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>*.xml</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="impBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(85, 255, 127);</string>
              </property>
              <property name="text">
               <string>导入</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="tableView"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextEdit" name="textEdit"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="text">
               <string>导入进度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QProgressBar{
background-color: rgb(127, 255, 0);
}

QProgressBar::chunk{
background-color:green;
width:10px;
margin:0.5px;
}</string>
              </property>
              <property name="value">
               <number>20</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(81, 170, 204, 255));
spacing:2px;
}
QToolButton
{
border:5px solid #008b8b;
border-radius:6px;
color:red;
}
QToolButton[popupMode=&quot;1&quot;]
{
padding-right:20px;
}

QToolButton:pressed
{
background-color: #20b2aa;
}

QToolButton:hover
{
border:5px solid #00ced1;
color:green;
}
</string>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_condb1"/>
   <addaction name="action_exp"/>
   <addaction name="action_sfwexit"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1236</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.0222727, x2:1, y2:1, stop:0.306818 rgba(37, 159, 0, 255), stop:0.943182 rgba(35, 223, 223, 255));
}

QMenuBar::item
{
spacing:3px;
padding:2px 4px;
border-radius:4px;
background-color:transparent;
}

QMenuBar::item:selected
{
color:#008080;
background-color:#e0ffff;
}

</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actiondakai"/>
    <addaction name="actionlingcunwei"/>
    <addaction name="actiontuichu"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionfuzhi"/>
    <addaction name="actionzhantie"/>
    <addaction name="actionjiantie"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_condb1">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/src/connectdb.jpg</normaloff>:/src/connectdb.jpg</iconset>
   </property>
   <property name="text">
    <string>连接数据库</string>
   </property>
   <property name="toolTip">
    <string>连接数据库</string>
   </property>
  </action>
  <action name="action_exp">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/src/imp.jpg</normaloff>:/src/imp.jpg</iconset>
   </property>
   <property name="text">
    <string>数据导出</string>
   </property>
   <property name="toolTip">
    <string>数据导出</string>
   </property>
  </action>
  <action name="action_sfwexit">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/src/exit.jpg</normaloff>:/src/exit.jpg</iconset>
   </property>
   <property name="text">
    <string>退出软件</string>
   </property>
   <property name="toolTip">
    <string>退出软件</string>
   </property>
  </action>
  <action name="actiondakai">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionlingcunwei">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actiontuichu">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionfuzhi">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionzhantie">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionjiantie">
   <property name="text">
    <string>剪贴</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
